examples/Nat/NAT.cs

```diff
@@ -42,17 +42,39 @@
 
 namespace Pax.Examples.Nat
 {
+  /// <summary>
+  /// A packet processor that performs Network Address Translation (a NAT).
+  /// </summary>
   public sealed class NAT : SimplePacketProcessor
```

Comment:
I normally seal all classes that aren't explicitly intended as superclasses, as it makes the intention clearer. Although `NAT` could be subclassed in the future, I have not made any allowances for it. Perhaps once the code-base is stable I should go through and check it and open `NAT`.